### PR TITLE
upgrade to 0.19

### PR DESCRIPTION
**Pull request by gyzerok**
Fix https://github.com/evancz/elm-sortable-table/issues/34.

This is mostly just automatic `elm-upgrade`. There are only 2 places I had to change because of shadowing, will highlight them in comments.

### CODE REVIEW

File: src/Table.elm [hunk 1]
```diff
@@ -175,87 +182,98 @@ summaries of various columns. And maybe you want to put attributes on `<tbody>`
...
-          , lightGrey "↕"
-          ]
-
-        Reversible (Just isReversed) ->
-          [ Html.text name
-          , darkGrey (if isReversed then "↑" else "↓")
-          ]
-  in
-    Html.th [ onClick ] content
+simpleTheadHelp ( name, status, onHeaderClick ) =
```
Comment (gyzerok) [inline src/Table.elm:229]: Here rename `onClick` to `onHeaderClick`

File: src/Table.elm [hunk 2]
```diff
@@ -506,44 +531,53 @@ viewCell data {viewData} =
...
-  case sorter of
-    None ->
-      data
+    case sorter of
+        None ->
+            data
 
-    Increasing sort ->
-      sort data
+        Increasing increasingSorter ->
```
Comment (gyzerok) [inline src/Table.elm:548]: In this function replace all occurences of `sort` with more specific names.

**Comment by evancz** [comment]
I think it does not make sense for this to be published under my name anymore. I think the best path is to have someone fork and publish their own thing for 0.19. They can respond to issues and feature requests however they see fit. They can even just make a totally new package that is how they'd prefer it.

**Comment by billstclair** [comment]
I missed this pull request before doing the 0.19 upgrade to [billstclair/elm-sortable-table](https://package.elm-lang.org/packages/billstclair/elm-sortable-table/latest).

**Comment by jhrcek** [comment]
@evancz Looks like @billstclair took over. Would you mind adding a note to (top of) the readme in this repo stating that this package won't be migrated to 0.19 (and maybe that people should use the forked version instead)? I was one of the happy users of this package and I think there will be more people like that.

**Comment by evancz** [comment]
I made a note about the situation.

**Pull request closed without merging**